syntax = "proto3";

package coherence;

service coherence_offload {
  rpc coherence_js (coherence_req) returns (coherence_answ) {}
}

message coherence_req {
  string req = 1;
}

message coherence_answ {
  string answ = 1;
}
